time-calculator/time_calculator.py

```diff
@@ -0,0 +1,67 @@
+
+def add_time(startTime, stopTime, dayOfWeek=None):
+
+    # Start Time
+    [startTime, start_am_pm] = startTime.split()
+    [startHours, startMinutes] = startTime.split(':')
+
+    # End Time
```

Comment:
1. same as above
2. Here the comment isn't matching the variable names below .... so use the name you think is better `stopTime/endTime`